finance_research/search/base.py

```
@@ -0,0 +1,585 @@
+                    if report_type == "annual"
+                    else f"{name} {period} 季报"
+                )
+                parts = [f"{name} {period} 营业收入 {use_rev:.2f} 亿元"]
+                if use_net is not None:
+                    parts.append(f"净利润 {use_net:.2f} 亿元")
+                if report_type == "annual":
+                    if gross_margin is not None:
+                        parts.append(f"毛利率 {gross_margin:.2f}%")
+                    if net_margin is not None:
+                        parts.append(f"净利率 {net_margin:.2f}%")
+                snippet = "，".join(parts) + "。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)